modin/backends/pandas/query_compiler.py

```diff
@@ -1898,6 +1898,8 @@ def insert(self, loc, column, value):
                 value = value.reindex(self.index)
             else:
                 value = list(value)
+        else:
+            value = [value] * len(self.index)
```

Comment:
I think it's better to leave a `value` creation with `np.full`.

Reply:
Currently, `np.full` is used in API layer and here all list-like structures (including `np.array`) are transformed to `list`. So I just keep semantics by building a list here.

Reply:
Okay, I see. Then you just need to rebase on master in order to rerun tests.